docs/ensnode.io/src/styles/starlight.css

```
@@ -372,3 +393,94 @@ ul.top-level details details > summary .group-label span {
+  border-bottom: none;
+
+  & a[role="tab"] {
+    padding-bottom: 8px;
+  }
+}
+
+/* Add custom styling to aside tip/caution/note components */
+.starlight-aside {
+  padding: 20px 16px;
+
+  border-radius: 12px;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)